packages/at_secondary_server/test/enroll_verb_test.dart

```diff
@@ -1154,13 +1155,13 @@ void main() {
       String unrevokeEnrollmentCommand = 'enroll:unrevoke:{"enrollmentId":""}';
       enrollVerbParams =
           getVerbParam(VerbSyntax.enroll, unrevokeEnrollmentCommand);
-      await expectLater(
+      expect(
           () => enrollVerbHandler.processVerb(
               response, enrollVerbParams, inboundConnection),
           throwsA(predicate((dynamic e) =>
               e is AtEnrollmentException &&
               e.message ==
-                  'enrollmentId is mandatory for enroll:revoke/enroll:deny')));
+                  'enrollmentId is mandatory for enroll:revoke/enroll:deny/enroll:delete')));
```

Comment:
If you make the change I suggested above then you meed to change this expect also

Reply:
Chang made in [3ca6f91](https://github.com/atsign-foundation/at_server/pull/2095/commits/3ca6f9125f3bb4abbf71a69232624c730f6460b6). Thanks for the heads-up.